### PR TITLE
ref(dynamic-sampling): Move draggable list item styles to the draggable list component

### PR DESCRIPTION
**Pull request by priscilawebdev**
. Move draggable item styles to the draggable list component folder
. Expose useful DraggableList Props

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/form.tsx [hunk 1]
```diff
@@ -259,7 +259,12 @@ class Form<P extends Props = Props, S extends State = State> extends React.Compo
             />
           )}
           <NumberField
-            label={t('Sampling Rate')}
+            label={
+              <React.Fragment>
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/form.tsx:263]: decided to push this small change in this PR

File: src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/form.tsx [hunk 2]
```diff
@@ -259,7 +259,12 @@ class Form<P extends Props = Props, S extends State = State> extends React.Compo
...
+                {t('Sampling Rate')}
+                {' \u0025'}
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/form.tsx:265]: Any reason to not use '%' on its own?

**Review by markstory — APPROVED** [review]
Looks good. Having the dragging styles with the item should help us reuse that style as we move more draggables over.

File: src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/form.tsx (hunk 2 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/modals/form.tsx:265]: no, I just wanted to be consistent with other punctuation marks that we have in our codebase

**Pull request merged**